pkg/commands/addpatch.go

```diff
@@ -70,8 +70,22 @@ func (o *addPatchOptions) Complete(cmd *cobra.Command, args []string) error {
 
 // RunAddPatch runs addPatch command (do real work).
 func (o *addPatchOptions) RunAddPatch(fsys fs.FileSystem) error {
-	if !fsys.Exists(o.patchFilePath) {
-		return errors.New(o.patchFilePath + " doesn't exist")
+	var patches []string
+
+	for _, pattern := range o.patchFilePaths {
+		files, err := fsys.Glob(pattern)
+		if err != nil {
+			return err
+		}
+		if len(files) == 0 {
+			log.Printf("%s has no match", pattern)
+			continue
+		}
+		patches = append(patches, files...)
+	}
+
+	if len(patches) == 0 {
+		return nil
```

Comment:
can we share the code in addresource that does the same thing?

fine with some private method in a pkg/commands/util.go file or something.

Reply:
Yes, we can. Let me update it.

Reply:
updated.